pkgs/development/python-modules/ffmpy/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, ffmpeg
+}:
+
+buildPythonPackage rec {
+  pname = "ffmpy";
+  version = "0.3.0";
+
```

Add an explicit format.

```suggestion
  format = "setuptools";
```